yorkie/src/main/kotlin/dev/yorkie/document/change/ChangePack.kt

```diff
@@ -0,0 +1,33 @@
+package dev.yorkie.document.change
+
+import dev.yorkie.document.time.TimeTicket
+
+/**
+ * [ChangePack] is a unit for delivering changes in a document to the remote.
+ *
+ * @property documentKey the key of the document.
+ * @property checkPoint It is used to determine the client received changes.
+ * @property snapshot a byte array that encodes the document.
```

Comment:
@hackerwins "s" ommited in [js version](https://github.com/yorkie-team/yorkie-js-sdk/blob/975d1b1540a54679c497771132ab99e7549f9c78/src/document/change/change_pack.ts#L40)
`encode` -> `encodes`